conductor-client/src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -244,6 +244,23 @@ class ExternalDatabaseManagementService(
         return organizationExternalDatabaseTables.getValue(tableId)
     }
 
+    fun getExternalDatabaseTableSchema(organizationId: UUID, tableId: UUID): String? {
+
+        val table = getOrganizationExternalDatabaseTable(tableId)
+        val sql = getExternalDatabaseTableSchemaSql(table.name)
+        val schemas = BasePostgresIterable(
+                StatementHolderSupplier(externalDbManager.connectToOrg(organizationId), sql)
+        ) { rs ->
+            rs.getString(PostgresColumn.PG_SCHEMA_NAME)
+        }.toList()
+
+        if (schemas.size != 1) {
```

Comment:
this seems unnecessary, unless you specifically want to return null in the case that there are multiple results

Reply:
That's the intention. I don't know how to handle that edge case, so returning null seemed the most reasonable.